### PR TITLE
Adds afva in all fcns flags (if any) when performing deeper analysis

### PR DESCRIPTION
**Pull request by AandersonL**
- [x] Mark this if you consider it ready to merge
- [x] I've added tests (optional)

When running `aaa` analysis in binaries, not all functions have their arguments analyzed which leaves an incomplete analysis.  This PR insures that if any `fcns` were found, it will have their arguments analyzed either.

### CODE REVIEW

**Comment by trufae** [comment]
please run r2r -i  test/db/anal/mips

**Comment by trufae** [comment]
And maybe only run this if anal.vars is set :?

File: libr/core/cmd_anal.c [hunk 1]
```diff
@@ -10780,6 +10780,13 @@ static int cmd_anal_all(RCore *core, const char *input) {
 			r_core_anal_all (core);
 			r_print_rowlog_done (core->print, oldstr);
 			r_core_task_yield (&core->tasks);
+			
+			// Run afvn in all fcns 
+			oldstr = r_print_rowlog (core->print, "Analyze all functions arguments/locals");
+			r_cons_break_push(NULL, NULL);
```
Comment (trufae) [inline libr/core/cmd_anal.c:10786]:
```suggestion
```

File: libr/core/cmd_anal.c [hunk 2]
```diff
@@ -10780,6 +10780,13 @@ static int cmd_anal_all(RCore *core, const char *input) {
...
+			r_core_cmd0(core, "afva @@ fcn.*");
```
Comment (trufae) [inline libr/core/cmd_anal.c:10787]:
```suggestion
			r_core_cmd0 (core, "afva @@ fcn.*");
```

File: libr/core/cmd_anal.c [hunk 3]
```diff
@@ -10780,6 +10780,13 @@ static int cmd_anal_all(RCore *core, const char *input) {
...
+			r_print_rowlog_done(core->print, oldstr);
```
Comment (trufae) [inline libr/core/cmd_anal.c:10788]:
```suggestion
			r_print_rowlog_done (core->print, oldstr);
```

File: libr/core/cmd_anal.c (hunk 1 above)
Comment (trufae) [reply libr/core/cmd_anal.c:10786]: dont push if you dont pop

**Review by trufae — CHANGES_REQUESTED** [review]

File: libr/core/cmd_anal.c [hunk 4]
```diff
@@ -10782,11 +10782,12 @@ static int cmd_anal_all(RCore *core, const char *input) {
 			r_core_task_yield (&core->tasks);
 			
 			// Run afvn in all fcns 
-			oldstr = r_print_rowlog (core->print, "Analyze all functions arguments/locals");
-			r_cons_break_push(NULL, NULL);
-			r_core_cmd0(core, "afva @@ fcn.*");
-			r_print_rowlog_done(core->print, oldstr);
-			
+			if (r_config_get_i (core->config, "anal.vars")) {
```
Comment (trufae) [inline libr/core/cmd_anal.c:10785]:
```suggestion
			if (r_config_get_b (core->config, "anal.vars")) {
```

Comment (AandersonL) [reply libr/core/cmd_anal.c]: done

File: libr/core/cmd_anal.c [hunk 5]
```diff
@@ -10780,6 +10780,14 @@ static int cmd_anal_all(RCore *core, const char *input) {
 			r_core_anal_all (core);
 			r_print_rowlog_done (core->print, oldstr);
 			r_core_task_yield (&core->tasks);
+			
+			// Run afvn in all fcns 
+			if (r_config_get_b (core->config, "anal.vars")) {
+				oldstr = r_print_rowlog (core->print, "Analyze all functions arguments/locals");
+				r_core_cmd0 (core, "afva @@ f");
```
Comment (trufae) [inline libr/core/cmd_anal.c:10787]:
```suggestion
				r_core_cmd0 (core, "afva @@f");
```

File: libr/core/cmd_anal.c [hunk 6]
```diff
@@ -10780,6 +10780,14 @@ static int cmd_anal_all(RCore *core, const char *input) {
 			r_core_anal_all (core);
 			r_print_rowlog_done (core->print, oldstr);
 			r_core_task_yield (&core->tasks);
+			
+			// Run afvn in all fcns
```
Comment (Lazula) [inline libr/core/cmd_anal.c:10784]: trailing whitespace

**Comment by Lazula** [comment]
Looks like afva throws out special naming for argc and argv. See test `db/cmd/cmd_att att varsub 2`, `argc` becomes `arg_8h` and `argv` becomes `arg_18h`. Seems undesirable, but this is more likely a problem with `afva` than these changes. Maybe that needs to be handled before we can just force `afva` every time.

**Comment by trufae** [comment]
i tried this https://github.com/radareorg/radare2/pull/19803

File: libr/core/cmd_anal.c (hunk 6 above)
Comment (trufae) [inline libr/core/cmd_anal.c:10784]:
```suggestion
			// Run afvn in all fcns
```

Comment (trufae) [reply libr/core/cmd_anal.c:10784]: will be good if github could show such artifacts in red background or sthg

Comment (Lazula) [inline libr/core/cmd_anal.c:10784]:
```suggestion

			// Run afvn in all fcns
```

Comment (Lazula) [reply libr/core/cmd_anal.c:10784]: line above the comment has whitespace too

**Comment by trufae** [comment]
i finished your pr in here https://github.com/radareorg/radare2/pull/19803

**Pull request closed without merging**